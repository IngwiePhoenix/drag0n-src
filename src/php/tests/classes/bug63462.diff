002+ 
003+ Notice: Undefined property: Test::$nonExisting in /Users/Ingwie/Work/drag0n-php/tests/classes/bug63462.php on line 17
002- Notice: Undefined index: nonExisting in %__set__get_006.php on line %d
004- Notice: Undefined index: publicProperty in %__set__get_006.php on line %d
005+ 
006+ Notice: Undefined property: Test::$publicProperty in /Users/Ingwie/Work/drag0n-php/tests/classes/bug63462.php on line 17
006- Notice: Undefined index: protectedProperty in %__set__get_006.php on line %d
008+ 
009+ Notice: Undefined property: Test::$protectedProperty in /Users/Ingwie/Work/drag0n-php/tests/classes/bug63462.php on line 17
008- Notice: Undefined index: privateProperty in %__set__get_006.php on line %d
011+ 
012+ Notice: Undefined property: Test::$privateProperty in /Users/Ingwie/Work/drag0n-php/tests/classes/bug63462.php on line 17