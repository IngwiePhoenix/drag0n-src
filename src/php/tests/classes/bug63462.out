__get nonExisting

Notice: Undefined property: Test::$nonExisting in /Users/Ingwie/Work/drag0n-php/tests/classes/bug63462.php on line 17
__get publicProperty

Notice: Undefined property: Test::$publicProperty in /Users/Ingwie/Work/drag0n-php/tests/classes/bug63462.php on line 17
__get protectedProperty

Notice: Undefined property: Test::$protectedProperty in /Users/Ingwie/Work/drag0n-php/tests/classes/bug63462.php on line 17
__get privateProperty

Notice: Undefined property: Test::$privateProperty in /Users/Ingwie/Work/drag0n-php/tests/classes/bug63462.php on line 17
__isset nonExisting
__isset publicProperty
__isset protectedProperty
__isset privateProperty
__set nonExisting
__set publicProperty
__set protectedProperty
__set privateProperty